lib.nix

```diff
@@ -96,7 +96,7 @@ let
       };
       fileToCall = builtins.toFile "extract-inputs.ncl" ''
         let contracts = import "${./.}/contracts.ncl" in
-        let nickel_expr = import "${sources}/${nickelFile}" in
+        let nickel_expr = import "${nickelFile}" in
```

Comment:
fixing `fixing ` > error: import of /nix/store/p6qc3j8vgfj1c918m04mg4hgsln4lwq0-c-hello-world//nix/store/i4bm1rmvjn37iz4fmn7bshwb3jgfg81x-hello.ncl failed: No such file or directory (os error 2)``